doc/user_doc/vic_installation/SUMMARY.md

```diff
@@ -0,0 +1,13 @@
+# Summary
+
+* [Introduction](README.md)
+* [Installing vSphere Integrated Containers](installing_vic.md)
+* [Environment Prerequisites for vSphere Integrated Containers Installation](vic_installation_prereqs.md)
+   * [Create a Private Port Group for Virtual Container Hosts](create_a_private_port_group_for_vch.md)
+* [Use the Vagrant DevBox VM to Build the vSphere Integrated Containers Binaries](set_up_devbox.md)
```

Comment:
Yes, now that the Windows and other versions are in the binaries, this is less necessary. I have removed this from the pub, but will leave the file in the repo.